pyuvdata/parameter.py

```diff
@@ -103,34 +104,47 @@ def __eq__(self, other):
             # only check that value is identical
             if not isinstance(self.value, other.value.__class__):
                 print(
```

Comment:
Just in general, do we want these to be print statements? Should they instead be warnings? I'm thinking that these equality checks are typically done inside of a `.check()` call rather than explicitly invoked by the user, so that might help raise awareness that there's something wrong. We can leave them as prints, just a thought I had.

Reply:
yeah, I've been wondering that for awhile. I think that's a big change and we should do it in another PR if we want to. Sometimes we expect non-equality, so a warning might be a little weird...

Reply:
These aren't actually called in the `check` method, they're called if you test objects for equality.

Reply:
Ah good point. And true, if we decide to change it we should do it in a different PR.